### PR TITLE
do_not_disturb: fix dunst's state=None

### PR DESCRIPTION
**Pull request by lasers**
This fixes `state = None` for dunst because we can query state from others, but not dunst.
* xfce - via querying -- return 0/1
* lxqt - via pgrepping -- is it running or not 
* dunst - no way to query its state

### CODE REVIEW

File: py3status/modules/do_not_disturb.py [hunk 1]
```diff
@@ -166,6 +166,8 @@ def post_config_hook(self):
                 self.backend.toggle(self.state)
             else:
                 raise Exception(STRING_INVALID_STATE.format(self.state))
+        else:
+            self.state = -1
```
Comment (maximbaz) [inline py3status/modules/do_not_disturb.py]: A better approach would be to raise the same error "invalid state" and let people know that when using dunst, it is not supported to set `state` to `None`, not introduce a third "unknown" state to the module.

```suggestion
            if self.server == "dunst":
                raise Exception(STRING_INVALID_STATE.format(self.state))
```

Comment (lasers) [reply py3status/modules/do_not_disturb.py]: >None: don't toggle anything on startup

This does that. The problem is just that we can't query anything from Dunst so this is accurate because it's not toggling anything. To give it `0` or `1` by default is misleading at best. If users does not like or want to change their state, albeit unknown, they still can change it there and the `degraded` would be gone.

If you revert 33a1654, then add this on top of it, then you can see how it is less biased / more honest. If `dunst` have a way to query state, then it would be done first thing first in main method (i.e. `self.state = self.backend.get_state()`).

tldr; Xfce, lxqt gets `-1` on `state = None` too, but will be replaced with a real value. Dunst just stays on `-1` untl we toggle something.

Comment (maximbaz) [reply py3status/modules/do_not_disturb.py]: I would argue that in the docstring:

```
state: specify state to use on startup
    None: don't toggle anything on startup
```

The description of None is the implementation detail and doesn't quite describe the state on startup, so the docstring better be adapted. 

In practice when user sets `state = None` in their config, what users really expect is py3status to query the current state from their notification manager. The comment should say: 

> None: query current state from notification manager (doesn't work on dunst).

Whenever https://github.com/dunst-project/dunst/issues/445 gets implemented, we will update the comment and stop throwing exception.

Comment (lasers) [reply py3status/modules/do_not_disturb.py]: Done.

**Comment by ultrabug** [comment]
Ok for me, waiting for Maxim's approval

**Comment by lasers** [comment]
@ultrabug Can you try checkout`c0f84dd` + cherrypick `dc0dbd3` and use this config...
```python
order += "do_not_disturb"
do_not_disturb {
    pause = True
    state = None
}
```
Sure, you get `degraded` only because we can't query Dunst. The state is always unknown at startup, but is consistent between i3 restarts.

If you find that this is accurate, I'm wondering because this is Dunst, maybe we could change default settings (like above) to favor Dunst more rather than all three notification servers because we probably have more users using Dunst than Xfce4-Notify anyway.

The real fix might be just to use a different default settings to favor Dunst more.

**Comment by ultrabug** [comment]
@lasers sorry but I don't have time to micro check this
